contracts/contracts/OneTimeSBT.sol

```diff
@@ -87,7 +87,7 @@ contract OneTimeSBT is ERC721Enumerable {
         for (uint256 i = 0; i < 3; i++) {
             revealedData_packed[i] = p_proof.pubSignals[PROVE_RSA_REVEALED_DATA_PACKED_INDEX + i];
         }
-        bytes memory charcodes = fieldElementsToBytes(
+        bytes memory charcodes = _fieldElementsToBytes(
```

Comment:
fieldElementsToBytes is only used here. So, I changed this function to internal and the name in _camelCase